src/index.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
from wiki, need write test with jest

Reply:
sorry a little confuse, the project is using Jest to do test. Do you mean we need write more test case?

Reply:
I mean no any jest functions you used in the test. Your current test doesn't have any valid check. I think should at least use `https://facebook.github.io/jest/docs/en/snapshot-testing.html#content` to check whether the result is same as expected.

And better test the valid check functions and submit button click function.  
Suggest to use jest + enzyme to simulate the actions and check the result, it is easier to add tests.

And better comment the coverage in pr comments. Then the reviewer is more clear for the test coverage.

Reply:
test submit function by dispatching action  store.dispatch(accountFormSubmit(some example JSON))